### DISCUSSION TITLE
Digitizer - AI on the edge - watermeter || Digitale & Analog ROI's Werte richtig erkennen

### DISCUSSION THREAD

**Discussion by Takahara2022** [jomjol/AI-on-the-edge-device#3153] [General]
### Issue

Hallo Community,

ich suche nach Hilfe bzgl. genauerer Ablesung von analogen ROI's des Wasserzählers Projekt "Al on the edge - water meter.
Die erkannten Wasserstände sind recht sprunghaft. Bin mir sicher, dass es noch Verbesserungspotential hinsichtlich der optischen Messwerterkennung gibt. Habe bereits einiges probiert.
- LED neu positioniert um Reflexionen zu mindern.
- Digital ROI's and Analog ROI's neu ausgerichtet. 

Den Wasserzähler kann ich nicht einfach tauschen, da er von meinem örtlichen Versorger bereit gestellt wird. 
Was konkret sollte ich versuchen um eine genauere Ablesung zu erreichen?

![2024-07-27 14_58_34-watermeter - AI on the edge](https://github.com/user-attachments/assets/1489b921-e555-4843-85fe-27ab032aaedf)

![2024-07-27 15_02_46-watermeter - AI on the edge](https://github.com/user-attachments/assets/036b548b-9995-4b77-b77f-e406326783da)

![2024-07-27 15_03_11-watermeter - AI on the edge](https://github.com/user-attachments/assets/c8b121e9-cd24-4012-9581-5f406b17b6d4)

![2024-07-27 15_03_34-Wasserzählerstände - Grafana](https://github.com/user-attachments/assets/77c2f840-4d67-4a2b-8d0a-2fa8ce320e4f)

![2024-07-27 15_04_25-watermeter - AI on the edge](https://github.com/user-attachments/assets/60d74b85-13f0-4746-9645-93d024ffdf2e)

Vielen Dank im Vorraus
Takahara

**Comment by SybexX** [comment]
versuch mal deine analogen ROIs so zu versetzen, dass der Mittelpunckt des Kreises auf der Drehachse des Zeigers ist.
ungefähr so:
![352727002-c8b121e9-cd24-4012-9581-5f406b17b6d4](https://github.com/user-attachments/assets/78ade30b-e8a0-4661-966e-7a1df0233327)

hast du die modele ana-cont_1208_s2_q.tflite oder ana-cont_1300_s2.tflite schon ausprobiert?

**Comment by Takahara2022** [comment]
> versuch mal deine analogen ROIs so zu versetzen, dass der Mittelpunckt des Kreises auf der Drehachse des Zeigers ist. ungefähr so: ![352727002-c8b121e9-cd24-4012-9581-5f406b17b6d4](https://private-user-images.githubusercontent.com/587201/352763192-78ade30b-e8a0-4661-966e-7a1df0233327.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIxNjIyMDAsIm5iZiI6MTcyMjE2MTkwMCwicGF0aCI6Ii81ODcyMDEvMzUyNzYzMTkyLTc4YWRlMzBiLWU4YTAtNDY2MS05NjZlLTdhMWRmMDIzMzMyNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyOFQxMDE4MjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jYjQ2NjAyY2RjYTRmMTFhY2VmZjA1YTI0MGE3NzhiNDA1YWM3YThhZTMwMjJlNTMzNTRhMGVkODE3MTEzZDZlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.y_eeux2NpsM3n0qIabMcBe0YklowqS3esqeSdVuQZ9E)
> 
> hast du die modele ana-cont_1208_s2_q.tflite oder ana-cont_1300_s2.tflite schon ausprobiert?

Das Modell ana-cont_1300_s2.tflite kann ich nicht auswählen. Werde mal das ana-cont_1208_S2_q-tflite probieren und die analogen ROI's entsprechend deiner Beschreibung verschieben und mich anschließend noch mal melden.

**Comment by SybexX** [comment]
ana-cont_1300_s2.tflite kannst du unter dem folgenden Link downloaden:
https://github.com/jomjol/neural-network-analog-needle-readout
Die Datei ana-cont_1300_s2.tflite einfach in den Ordner "config" kopieren und dann taucht es in der Auswahl auf.
[analog_digital_models_28_07_2024.zip](https://github.com/user-attachments/files/16403024/analog_digital_models_28_07_2024.zip)
